rcode-reader/src/main/java/eu/rssw/pct/elements/MethodParameter.java

```diff
@@ -0,0 +1,104 @@
+package eu.rssw.pct.elements;
+
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+
+import eu.rssw.pct.DataType;
+import eu.rssw.pct.IParameter;
+import eu.rssw.pct.ParameterMode;
+import eu.rssw.pct.ParameterType;
+import eu.rssw.pct.RCodeInfo;
+
+public class MethodParameter extends ClassElement implements IParameter {
+  private static final int PARAMETER_APPEND = 1;
+  private static final int PARAMETER_HANDLE = 2;
+  private static final int PARAMETER_BIND = 4;
+
+  private int extent;
+  private int crc;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "crc" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1068)